maps/tether/tether_turfs.dm

```diff
@@ -4,12 +4,14 @@ VIRGO3B_TURF_CREATE(/turf/simulated/open)
 	edge_blending_priority = 0.5 //Turfs which also have e_b_p and higher than this will plop decorative edges onto this turf
 /turf/simulated/open/virgo3b/New()
 	..()
-	outdoor_turfs.Add(src)
+	if(outdoors)
+		outdoor_turfs.Add(src)
 
 VIRGO3B_TURF_CREATE(/turf/simulated/floor)
 /turf/simulated/floor/virgo3b/New()
 	..()
-	outdoor_turfs.Add(src)
+	if(outdoors)
```

Comment:
I think this is redundant with `/turf/simulated/floor/New()` and will double-add.

Reply:
You are correct. I had no idea that was even a thing. I guess I independently came up with that idea.